### ISSUE TITLE
Masking overlap

### ISSUE THREAD

**Issue by martin-rdz** [PollyNET/Pollynet_Processing_Chain#271]
When the values of the overlap function get too small (e.g. 1e-5), the overlap corrected products explode for the lowest bins.

![grafik](https://github.com/PollyNET/Pollynet_Processing_Chain/assets/31452472/8c5fe168-699c-438d-aba4-c9339b6d9cb3)

A simple mask before the overlapCorrection helps to address this:
```
olFunc355Raman(olFunc355Raman <= 0.01) = NaN;
olFunc532Raman(olFunc532Raman <= 0.01) = NaN;
```

**Comment by HolgerPollyNet** [comment]
yes, good point , but I am not happy with the flagging, instead the overlap function itself shall be flagged or improved.

But maybe something for the QC products (quality controlled products), we want to implement.
